stock_account_quantity_history_location/wizards/stock_quantity_history.py

```diff
@@ -12,6 +12,14 @@ class StockQuantityHistory(models.TransientModel):
     _inherit = "stock.quantity.history"
 
     def open_at_date(self):
+        def convert_domain(domain):
```

Comment:
For what is this needed?

Reply:
For the alternative but non performant approach I was trying to use `product.product._get_domain_locations()`. I needed to add an extra level for using it in the stock valuation layers.

A quick fix would be to renounce to child locations in this context and add the simple domain as shown